pkg/security/probe/probe_ebpfless.go

```diff
@@ -302,6 +304,7 @@ func (p *EBPFLessProbe) handleSyscallMsg(cl *client, syscallMsg *ebpfless.Syscal
 		p.processKiller.HandleProcessExited(event)
 	}
 
+	event.ResolveFields()
```

Comment:
Why do we need to call the ResolveFields here ? This function forces the resolutions of all the fields even if not used by any rules.

Reply:
ResolveFields removed in last commit